desc/imports.go

```diff
@@ -117,6 +123,9 @@ func (r *ImportResolver) ResolveImport(source, importPath string) string {
 			return res
 		}
 	}
+	if r.SkipFallbackRules {
```

Comment:
The tests are failing because you need to be able to deal with `r == nil` I believe. I think you need to change this to:

```
if r != nil && r.SkipFallbackRules {
```

Reply:
https://travis-ci.org/jhump/protoreflect/jobs/536066319

Reply:
Yep. Silly mistake from me not running the tests locally before pushing...

Reply:
Hey happens to all of us :-)